### DISCUSSION TITLE
Being able to adjust priorities for warrior.

### DISCUSSION THREAD

**Discussion by jrd-derooij** [Hekili/hekili#935] [Q&A] [answered]
Hey,

Not sure if I should post this here, or perhaps make it an issue. But I've been trying to get a entry in my priority list for fury warrior where I want to cancel the bladestorm aura if certain conditions are met (condemn is usable and we're in execute phase). But for some reason it seems I can't get Bladestorm to show up as an buff that can be canceled. All the code seems to be there, when I hover over it, I see that it shows all of the conditions to be true, but then the 'shown' value stays false. 

If I try the same thing but make it show that I have to cancel battle shout for instance, it does show. Is this something others might experience as well? Or am I doing something wrong here? :)

**Comment by Hekili** [comment, marked as answer]
This would be more appropriate for an issue ticket, with all of the required information for an issue ticket supplied.

**Reply by jrd-derooij** [reply]
I actually got it to work. I have no clue why this didn't work before now, but perhaps I still made a mistake in the configuration. It works now ;)
